runtime/src/commitment.rs

```diff
@@ -215,6 +216,61 @@ impl CommitmentSlots {
     }
 }
 
+#[derive(Default, Clone)]
+pub struct CommitmentBanks {
+    /// The slot of the bank from which all other slots were calculated.
+    pub bank: Arc<Bank>,
+    /// The current node root
+    pub root: Option<Arc<Bank>>,
```

Comment:
Ideally, we could get rid of all these `Option`s, but that would require changing a whole lot of code in ReplayStage to plumb banks through instead of slots.